templates/store/snap-distro-install.html

```diff
@@ -221,6 +221,16 @@ <h4 class="p-card__title">
   <script src="{{ static_url('js/dist/public.js') }}"></script>
   <script>
     Raven.context(function () {
+      try {
+        document.querySelector(".js-install")
+          .addEventListener('click', function (event) => {
+            event.preventDefault();
+            snapcraft.public.scrollTo("#install", 300, -20);
```

Comment:
Do we need to scroll to this instead of jump? If this to add the offset for a sticky header?

Reply:
@anthonydillon It was a UX request to have animated scroll. Offset is so that the card that we scroll to is not sticking to the top of the window, but has nice margin.

Reply:
Would be nice to add `#install` to the url and auto-jump to the anchor on page load.
Also could we make the "scrollTo" click event more generic?
```
try {
  snapcraft.public.scrollTo(link, {options}); //options with nice defaults, including duration and offset
} catch...
```

`scrollTo` would then:
- Figure out if link exists and fail fast
- Attach the click handler
- Figure out the element to jump to based on the `href` (and check that exists)
- Currently (if I'm reading right) `scrollTo` won't check if the `#target` exists and won't fallback to anchor tags?